examples/time_frequency/plot_source_tfr_power_and_itc.py

```diff
@@ -0,0 +1,105 @@
+"""
+========================================================================
+Compute induced power and itc in source space using dSPM and multitapers
+========================================================================
+
+Compute induced power and inter-trial-coherence in source space,
+using a multitaper time-frequency transform on a list of source
+estimate objects.
+
+"""
+# Authors: Dirk Gütlin <dirk.guetlin@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import numpy as np
+
+import mne
+from mne import io
+from mne.datasets import sample
+from mne.minimum_norm import read_inverse_operator, apply_inverse_epochs
+from mne.time_frequency import tfr_multitaper
+
+print(__doc__)
+
+###############################################################################
+# Set parameters
+data_path = sample.data_path()
+raw_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
```

Comment:
I would not use sample for such an example but rather somato. It contains real induced power.